blog/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
```

Comment:
may be adding test cases would give us a lot of confidence on this library

Reply:
@Pradhu-MAN Thanks for your feedback 
I will Update the PR with your required changes